src/client/theme-default/components/VPHero.vue

```diff
@@ -322,6 +322,8 @@ const heroImageSlotExists = inject('hero-image-slot-exists') as Ref<boolean>
   left: 50%;
   max-width: 192px;
   max-height: 192px;
+  height: 100%;
+  aspect-ratio: 1;
```

Comment:
aspect-ratio of images might not be 1

```suggestion
  width: 100%;
  height: 100%
  object-fit: contain;
```